util/disk-image-validator/README.md

```diff
@@ -0,0 +1,152 @@
+# Disk Image Validator
+
+This utility is used to verify that full system workloads in gem5 function as expected. It checks:
+
+- That expected exit events occur in the correct order
+- The `gem5-bridge` driver is properly installed and functioning
+- The workload can at least boot with various configurations
+
+## gem5-Bridge Driver Validator
+
+To validate the `gem5-bridge` driver, run the `gem5-bridge-driver-validate.py` script. This script verifies the driver installation by executing a simple C program that makes an m5 hypercall (hypercall number 8), without requiring superuser privileges.
+
+### Usage
+
+```bash
+build/ALL/gem5.opt util/disk-img-validator/gem5-bridge-driver-validate.py --isa <ISA> --workload <WORKLOAD_ID> --resource-version <RESOURCE_VERSION>
```

Comment:
this is a chicken and egg problem: why would you create a resource before validating the disk image? Yet this validator requires a resource to be passed for validation. Creating a resource from a disc image has a time coat and a little annoying when you just want to literally test a disk you created for is valid 

Is there anyway we could have this so it accepts a disk image and kernel so people can validate their disc image upon creation